neurogym/wrappers/monitor.py

```diff
@@ -34,39 +37,35 @@ class Monitor(Wrapper):
 
     def __init__(
         self,
-        env,
-        folder=None,
-        sv_per=100000,
-        sv_stp="trial",
-        verbose=False,
-        sv_fig=False,
-        num_stps_sv_fig=100,
-        name="",
-        fig_type="png",
-        step_fn=None,
+        env: ngym.TrialEnv,
+        conf: Conf | str | Path | None = None,
+        step_fn: Callable | None = None,
```

Comment:
Thanks for adding type hints here. Don't forget to update the docstring above as well.

Do we want to allow `conf` being a string? seems like `Conf` and `Path` should be enough options, no?

While we're fixing this up, it would be good to also improve the variable names to `step_function` or something clear like that.

Reply:
> Do we want to allow conf being a string

I don't see why not - we can load a configuration passed as a string. It's just for documentation anyway, the user can try to pass whatever they like 🙂